examples/step-61/step-61.cc

```diff
@@ -441,7 +441,7 @@ namespace Step61
     // @p face component of the shape functions.
     const FEValuesExtractors::Vector velocities(0);
     const FEValuesExtractors::Scalar interior(0);
-    const FEValuesExtractors::Scalar face(1);
+    const FEValuesExtractors::Scalar face_extractor(1);
```

Comment:
Let's find a better name for this than to duplicate `_extractor`. If you call the previous variable `p_interior`, then this could be called `p_interface`.